bindings/wasm/examples/src/0_basic/6_create_vp.ts

```diff
@@ -188,13 +188,16 @@ export async function createVP() {
     // Validate the credentials in the presentation.
     let credentialValidator = new JwtCredentialValidator();
     let validationOptions = new JwtCredentialValidationOptions({
-        subjectHolderRelationship: [presentationHolderDID.toString(), SubjectHolderRelationship.AlwaysSubject],
+        subjectHolderRelationship: [
+            presentationHolderDID.toString(),
+            SubjectHolderRelationship.AlwaysSubject,
+        ],
```

Comment:
Nit:

- There are five instances of `new JwtCredentialValidationOptions({})` in this and other examples that you could rewrite to `new JwtCredentialValidationOptions()`. Search for `({})` in the examples directory.
- And we can rewrite all instances of `JwtCredentialValidationOptions.default()` to `new JwtCredentialValidationOptions()` as well in the examples.

We could consider removing the `default` method on `{JwtCredentialValidationOptions, JwtPresentationValidationOptions, JwtPresentationOptions}` and make the `new`-style pattern the default.

(Feel free to do in another PR as well, I just looked it up now and wanted to write it down.)